api/instance/v1/instance_sdk.go

```diff
@@ -1531,7 +1531,7 @@ type DeleteServerRequest struct {
 
 // DeleteServer delete server
 //
-// Delete a server with the given ID.
+// Delete a server with the given ID. Be wary that when terminating a server, all the attached volumes (local *and* block storage) are deleted. So, if you want to keep your local volumes, you must use the `archive` action instead of `terminate`. And if you want to keep block-storage volumes, **you must** detach it beforehand you issue the `terminate` call.  For more information, read the [Volumes](#volumes-7e8a39) documentation.
```

Comment:
This documentation does not belong here but on the `DoAction` method or the `ActionTerminate` enum value